src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -80,7 +83,17 @@ public class RepositoryCloneViewModel : BaseViewModel, IRepositoryCloneViewModel
                 signalReset: filterResetSignal
             );
 
-            BaseRepositoryPathValidator = this.CreateBaseRepositoryPathValidator();
+            var baseRepositoryPath = this.WhenAny(
+                x => x.BaseRepositoryPath,
+                x => x.SelectedRepository,
+                (x, y) => x.Value);
+
+            BaseRepositoryPathValidator = ReactivePropertyValidator.ForObservable(baseRepositoryPath)
+                .IfNullOrEmpty("Please enter a repository path")
+                .IfTrue(x => x.Length > 200, "Path too long")
+                .IfContainsInvalidPathChars("Path contains invalid characters")
+                .IfPathNotRooted("Please enter a valid path")
+                .IfTrue(IsAlreadyRepoAtPath, Resources.RepositoryNameValidatorAlreadyExists);
```

Comment:
The issue was that in the create view model the repository name is validated separately from the base path, whereas in the clone dialog we don't need to validate the name as it's already coming from GitHub. So the options were either merge the two validators in the create view model somehow or have two validators for a single textbox in clone. If that makes sense?